test/business/badgeCalculator.js

```diff
@@ -1,48 +0,0 @@
-// Copyright (c) The Linux Foundation and others. Licensed under the MIT license.
-// SPDX-License-Identifier: MIT
-
```

Comment:
This test is still relevant. it just becomes a test of `definitionService.computeScore`